docs/zora-network/network.mdx

```diff
@@ -66,23 +66,21 @@ Zora Sepolia Network is an Testnet L2 built on top of Ethereum Sepolia using the
 
 | Name                                | Address                                    |
 | ----------------------------------- | ------------------------------------------ |
-| L1ERC721Bridge                      | 0xDBCdA21518AF39E7feb9748F6718D3db11591461 |
-| SystemDictator                      | 0x2E44e62992f14b904Bfefd93e63D98D7dA4fcD66 |
-| PortalSender                        | 0xd6C5Df0a29562521b2B26fAc218e3dAf0a4dFC9B |
-| L1StandardBridge                    | 0xbF6acaF315477b15D638bf4d91eA48FA79b58335 |
-| Lib_AddressManager                  | 0xEF8115F2733fb2033a7c756402Fc1deaa56550Ef |
-| L2OutputOracleProxy                 | 0x9E6204F750cD866b299594e2aC9eA824E2e5f95c |
-| OptimismMintableERC20FactoryProxy   | 0xc52BC7344e24e39dF1bf026fe05C4e6E23CfBcFf |
-| Proxy\_\_OVM_L1StandardBridge       | 0x3e2Ea9B92B7E48A52296fD261dc26fd995284631 |
-| ProxyAdmin                          | 0xD4ef175B9e72cAEe9f1fe7660a6Ec19009903b49 |
-| OptimismMintableERC20Factory        | 0x84ee4b9673598ca2FbDad4Ba4a27A58D6328Ec46 |
-| OptimismPortal                      | 0x43260ee547c3965bb2a0174763bb8FEcC650BA4A |
-| Proxy\_\_OVM_L1CrossDomainMessenger | 0xdC40a14d9abd6F410226f1E6de71aE03441ca506 |
-| L1ERC721BridgeProxy                 | 0x83A4521A3573Ca87f3a971B169C5A0E1d34481c3 |
-| SystemConfig                        | 0x17fb7c8Ce213F1A7691ee41EA880ABf6eBC6fa95 |
-| OptimismPortalProxy                 | 0x1a0ad011913A150f69f6A19DF447A0CfD9551054 |
-| L1CrossDomainMessenger              | 0x363B4B1ADa52E50353f746999bd9E94395190d2C |
-| L2OutputOracle                      | 0x89336159Edd615260a95309e46343602D6b6489e |
-| SystemDictatorProxy                 | 0x50bA02c0Ca5E7bF772913EaF24Fb1fd3842a5f19 |
-| SystemConfigProxy                   | 0xA3cAB0126d5F504B071b81a3e8A2BBBF17930d86 |
+| OptimismMintableERC20FactoryProxy   | 0x5F3bdd57f01e88cE2F88f00685D30D6eb51A187c |
+| OptimismMintableERC20Factory        | 0x42D4a53fdcbc738479aaaC9a39e3eFe6A3216Bef |
```

Comment:
@isaaczaak i'm not seeing these contracts show up on the explorer. for example https://sepolia.explorer.zora.energy/address/0xE17071F4C216Eb189437fbDBCc16Bb79c4efD9c2

Reply:
this particular contract and many others are on sepolia: 
https://sepolia.etherscan.io/address/0x42D4a53fdcbc738479aaaC9a39e3eFe6A3216Bef

Reply:
this is for zora network no?

Reply:
The l1 part of the zora network is on sepolia.